app/Http/Controllers/UserController.php

```diff
@@ -67,4 +69,16 @@ public function updateUser(updateUserRequest $request, User $user)
     {
         $this->userService->updateUser($request->all(), $user);
     }
+
+    public function getUserEmails()
```

Comment:
let's name this action 'index'. It's the common name for an action that returns a collection of entities. In our case, we get a collection of users.
Each controller in laravel application have by default actions:
1) index  - to get collection
2) show - to get one specific entity
3) store - to create entity
4) update - to update specific entity
5) delete - to remove specific entity